scripts/mkcloud

```diff
@@ -757,9 +769,52 @@ function testsetup()
     onadmin testsetup
     local ret=$?
     $scp root@$adminip:tempest.log "$artifacts_dir/tempest.log"
+
+    # Register the cloud in Rally and import the results
+    if [ -n "$rally_server" ]; then
+        local title=$(testname)
+        local type=$(testtype)
+
+        $scp root@$adminip:.openrc "$artifacts_dir/.openrc"
+        scp_password "$artifacts_dir/tempest.log" rally@$rally_server:/tmp/tempest.$$.log
+        scp_password "$artifacts_dir/.openrc" rally@$rally_server:/tmp/.openrc.$$
+        ssh_password rally@$rally_server "
```

Comment:
IMHO, we should prefer to use ssh-keys instead of weak public default passwords.